libtiledbsoma/src/CMakeLists.txt

```diff
@@ -208,6 +216,12 @@ install(FILES
   DESTINATION "include/tiledbsoma/utils"
 )
 
+install(FILES
```

Comment:
Why does the reindexer and khash headers need to be installed? If we want to expose the reindexer there should be a clean public facing API so the internal class can stay abstracted from direct interaction.